src/server/lib/user-features.ts

```diff
@@ -1,13 +1,20 @@
 import { Response } from 'express';
 import { getConfiguration } from '@/server/lib/getConfiguration';
 import { getUserAttributes } from '@/server/lib/members-data-api/user-attributes';
+import { getUserBenefits } from './user-benefits-api/user-benefits';
+import { UserBenefitsResponse } from '@/shared/lib/user-benefits-api';
 
 // port of some functionality from https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/commercial/user-features.ts
 
 const { baseUri } = getConfiguration();
 
 const domain = `${baseUri.replace('profile.', '').split(':')[0]}`;
 
+export const AD_FREE_USER_COOKIE = 'GU_AF1';
+export const HIDE_SUPPORT_MESSAGING_COOKIE = 'gu_hide_support_messaging';
+export const ALLOW_REJECT_ALL_COOKIE = 'gu_allow_reject_all';
+export const USER_BENEFITS_EXPIRY_COOKIE = 'gu_user_benefits_expiry';
```

Comment:
```suggestion
const AD_FREE_USER_COOKIE = 'GU_AF1';
const HIDE_SUPPORT_MESSAGING_COOKIE = 'gu_hide_support_messaging';
const ALLOW_REJECT_ALL_COOKIE = 'gu_allow_reject_all';
const USER_BENEFITS_EXPIRY_COOKIE = 'gu_user_benefits_expiry';
```

I don't think these need to be an export?